src/Components/SmartComponents/Reports/Common/tablePage.js

```diff
@@ -46,9 +48,25 @@ const tablePage = ({ data, page, intl, header, type, isReportDynamic = false })
             hit: 'advisory_available'
         }
     ];
+    if (isAppliesToOSEnabled) {
+        POSSIBLE_HEADERS.splice(4, 0, {
+            name: 'rhel_version',
+            header: { value: intl.formatMessage(messages.appliedToOS) },
+            hit: 'rhel_versions'
+        });
+    }
+
```

Comment:
Why so complicated? Can't this be directly within defining the array and just add or not add the item to the array?

Reply:
@bastilian was trying to do that but if FF was set to false like this 
```
        isAppliesToOSEnabled && {
            name: 'rhel_version',
            header: { value: intl.formatMessage(messages.appliedToOS) },
            hit: 'rhel_versions'
        },
 ```
 if feature flag was false, item has been added but it was empty and because of that PDF columns looked improperly

Reply:
Ah! You need to spread it, like this:

```js
[
  ...isAppliesToOSEnabled ? [{
            name: 'rhel_version',
            header: { value: intl.formatMessage(messages.appliedToOS) },
            hit: 'rhel_versions'
        }] : [],
]
```

Reply:
Basically like you do on line 65. 😊

Reply:
@bastilian I was getting this but we want to avoid having item at all so columns look arranged properly
<img width="471" alt="Screenshot 2025-04-29 at 13 51 28" src="https://github.com/user-attachments/assets/3f9519c8-23b6-4f70-b09b-e9735ea79b7f" />

Reply:
@bastilian good catch ❤️ gonna fix it